apps/judicial-system/web/src/routes/Prosecutor/Indictments/Processing/Processing.tsx

```diff
@@ -76,6 +76,8 @@ const Processing: FC = () => {
     deleteCivilClaimant,
   } = useCivilClaimants()
   const router = useRouter()
+
+  // TODO: currently for e2e tests, this is failing locally due to this feature flag
```

Comment:
Sidenote: Discovered this when debugging the test locally, but should be resolved in follow-up PRs but added this if others see a failing tests locally 👍